src/renderer/src/components/Onboarding/OnboardingTopMenu.tsx

```diff
@@ -82,37 +81,53 @@ export function OnboardingTopMenu({ currentStep }: OnboardingTopMenuProps) {
 		3: 'CreateJoinProjectScreen',
 	}
 
+	const canGoBack = currentStep < 3
+
 	return (
 		<MenuContainer>
 			<GoBackButton
-				onClick={() => router.history.back()}
-				variant="text"
-				style={{
-					color: BLUE_GREY,
-					gap: 8,
-					padding: '12px 32px',
+				onClick={() => {
+					if (canGoBack) {
+						router.history.back()
+					}
 				}}
+				variant="text"
 				aria-label={formatMessage(m.goBack)}
+				disabled={!canGoBack}
+				sx={{
+					'&.Mui-disabled': {
+						color: BLUE_GREY,
+						opacity: 0.5,
+					},
+				}}
 			>
 				<BackArrow aria-hidden="true">←</BackArrow>
 				{formatMessage(m.goBack)}
 			</GoBackButton>
 			<Steps>
-				{[1, 2, 3].map((step) => (
-					<StepsContainer key={step}>
-						<Step
-							active={currentStep === step}
-							onClick={() =>
-								navigate({ to: `/Onboarding/${stepToRoute[step]}` })
-							}
-						>
-							<Text kind="body" bold={currentStep === step}>
-								{formatMessage(m.step, { number: step })}
-							</Text>
-						</Step>
-						{step < 3 && <Divider />}
-					</StepsContainer>
-				))}
+				{[1, 2, 3].map((step) => {
+					const isDisabled =
+						step > currentStep || (currentStep === 3 && step < currentStep)
```

Comment:
as mentioned in slack. lets just make none of these clickable as this logic is a bit confusing